doc/rho.1

```diff
@@ -42,71 +40,106 @@ The following sections cover these commands in more detail.
 .PP
 The primary purpose of rho is to scan a network. This can be done using simply 'rho scan' using network and authentication profiles.
 .PP
-By default, the authentication credentials, profiles created using rho are stored in an encrypted files. The files are encrypted with AES-128 encryption and is decrypted when the 'rho' command is run, using a vault password to access the file.
+By default, the authentication credentials, profiles created using rho are stored in encrypted files. The files are encrypted with AES-256 encryption and is decrypted when the 'rho' command is run, using a vault password to access the file.
 
 
 .SS AUTHENTICATION
-The first part to configuring rho is setting up authentication credentials. rho uses SSH to connect to the servers on the network, and the credentials is uses are configured in auth entries. Some kind of auth entry has to be passed when the scan is run, either by referencing the auth entry in a profile or by passing the credentials with the scan command.
+The first part to configuring rho is setting up authentication credentials. rho uses SSH to connect to the servers on the network, and the credentials used are configured in auth entries. An auth entry is passed when the scan is run by reference in a profile.
 .PP
 There can be multiple auth credentials contained in a single profile.
 .PP
 .B
-Creating and Editing auth Entries
+Creating and Editing Auth Entries
```

Comment:
I see _Credentials_ and _Auth Entries_ being used to mean the same thing. Are thos exchangeable or should we use the same term?

Reply:
I'm going to change this to uses **Authentication Credentials** through out the doc to try and remove the confusion.